OCPBUGS16018: CMA cannot be used with multi namespaces but the doc says we can (#73557)
openshift/openshift-docs: modules/nodes-cma-autoscaling-custom-install.adoc

        maxSize: "50"
  serviceAccount: {}
----

> <1> Specifies a single namespace where the custom autoscaler is to scale applications. Omit or set empty to scale applications in all namespaces. This field should have one namespace or be empty. The default is empty.

Reviewer: ```suggestion
<1> Specifies a single namespace in which the custom autoscaler should scale applications. Leave it blank or leave it empty to scale applications in all namespaces. This field should have a namespace or be empty. The default value is empty.
```
